core/src/main/java/org/mule/runtime/core/internal/config/togglz/scope/type/MuleTogglzFeatureScopeType.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.internal.config.togglz.scope.type;
+
+/**
+ * Mule Togglz Feature Scope Types.
+ *
+ * @see org.mule.runtime.core.internal.config.togglz.scope.MuleTogglzFeatureScope
+ */
+public enum MuleTogglzFeatureScopeType {
+
+  /**
+   * Represents a type of {@link org.mule.runtime.core.internal.config.togglz.scope.MuleTogglzFeatureScope} that corresponds to an
+   * artifact.
+   */
+  ARTIFACT_SCOPE_TYPE,
+
+  /**
+   * Represents a type of {@link org.mule.runtime.core.internal.config.togglz.scope.MuleTogglzFeatureScope} that corresponds to
+   * the whole runtime.
+   */
+  RUNTIME_SCOPE_TYPE;
```

Comment:
does this mean that there is a runtime level service for handling this?

Reply:
Yes, this is managed at runtime level (even the ones associated to an application scope)